src/builder-context.c

```diff
@@ -344,6 +346,28 @@ builder_context_find_in_sources_dirs_va (BuilderContext *self,
   return NULL;
 }
 
+void 
+builder_context_set_opt_export_only (BuilderContext *self,
+                                          gboolean       opt_export_only)
+{
+  self->opt_export_only = opt_export_only;
+}
+
+gboolean builder_context_get_opt_export_only (BuilderContext *self){
+  return self->opt_export_only;
+}
+
+void
+builder_context_set_opt_mirror_screenshots_url (BuilderContext *self,
+                                                     char *opt_mirror_screenshots_url){
+  self->opt_mirror_screenshots_url=opt_mirror_screenshots_url;
+}
+
+char *
+builder_context_get_opt_mirror_screenshots_url (BuilderContext *self){
+  return self->opt_mirror_screenshots_url;
+}
+
```

Comment:
Nitpick about weird formatting throughout here. `{` should be on new lines. params should be aligned.